tests/test_analysis.py

```diff
@@ -104,6 +132,7 @@ def test_get_pca_dimensions(self):
         single_num_dims = 1
 
         VAF, num_dims, proj_data = aopy.analysis.get_pca_dimensions(single_dim_data)
```

Comment:
could you add a call to get_pca_dimensions() with all the optional arguments to prove that they actually do something now